server/cmd/mmctl/commands/post_e2e_test.go

```diff
@@ -65,7 +65,7 @@ func (s *MmctlE2ETestSuite) TestPostListCmd() {
 
 		err := postListCmdF(s.th.Client, cmd, []string{teamName + ":" + channelName})
 		s.Require().NotNil(err)
-		s.Require().Contains(err.Error(), "You do not have the appropriate permissions.")
+		//s.Require().Contains(err.Error(), "You do not have the appropriate permissions.")
```

Comment:
I assume this needs reviewing.

Reply:
Yeah, I think we can just remove these altogether, since to me if you're trying to access a channel without permissions, it should return a 404 since we don't want to leak that the channel even exists if there's no permissions. I don't think we need to explicitly check for the error.

Thoughts?

Reply:
Checking for a specific error message has some value. Of course that can be a not found error.